include/deal.II/base/data_out_base.h

```diff
@@ -1658,6 +1658,11 @@ namespace DataOutBase
      */
     deal_II_intermediate,
 
+    /**
+     * Output in parallel deal.II intermediate format.
+     */
+    parallel_deal_II_intermediate,
```

Comment:
Hm, you make a good point. Not that hdf5 is listed in there as well and it can not be used with ``write()``. I can remove it for now. Alternatively, I could implement writing this new format also without MPI and support that in write(). Not sure how useful that is, though.

Reply:
Remove it for now. That might be easiest.